src/lib.rs

```diff
@@ -55,15 +55,20 @@ impl Configuration {
             .map(str::to_string)
             .collect::<Vec<String>>();
 
+        // I noticed you use a different approach with cache expiration and default quality.
+        // You take in an option and do the defaulting in the run function instead of like
+        // the others where you do the defaulting in main and send a non-Option value as
+        // the arguments to run. Up to you, but I think the approach of doing the optional
+        // unwrapping and defaulting in main makes the intent more obvious and erases the
+        // Option wrapper type from the run function signature.
```

Comment:
That's a good point — I'll move the logic to handle defaults into `main`

Reply:
Made that change here: https://github.com/walterbm/rusty-resizer/commit/70e88b0ff84d0737cec0fc91c4ecfe1e0fd06e73